diff --git a/node_modules/anychart-nodejs/lib/anychart-node.js b/node_modules/anychart-nodejs/lib/anychart-node.js
index 4c8c3e0..e6f5daa 100644
--- a/node_modules/anychart-nodejs/lib/anychart-node.js
+++ b/node_modules/anychart-nodejs/lib/anychart-node.js
@@ -152,7 +152,7 @@ function getBBox() {
   var fontStyle = this.getAttribute('font-style');
   if (fontStyle) fontStyle = fontStyle.toLowerCase();
 
-  var fontsArr = fontFamily.split(', ');
+  var fontsArr = (fontFamily||"").split(', ');
 
   var font;
   for (var i = 0, len = fontsArr.length; i < len; i++) {
